pages/invites.tsx

```diff
@@ -0,0 +1,127 @@
+//  import DashboardLayout from "components/DashboardLayout";
+import { useEffect, useState } from "react";
+// import deleteAcceptButton from "pages/declineButton";
+import DeclineButton from "components/declineButton";
+import AcceptButton from "components/acceptButton";
+
+const UserRequestDashboardItem: React.FunctionComponent<User> = ({
+  name,
+  email,
+  phoneNumber,
+}) => {
+  return (
+    <div className="d-flex flex-row">
+      <div className="flex flex-row justify-between text-sm text-center">
+        <div className="d-flex flex-row">{name}</div>
+        <div className="d-flex flex-row">{email}</div>
+        <div className="d-flex flex-row">{phoneNumber}</div>
+        <div className="d-flex flex-column">
+          <DeclineButton />
+        </div>
+        <div className="d-flex flex-column">
+          <AcceptButton />
+        </div>
+      </div>
+      <hr className="border-unselected border-opacity-50" />
+    </div>
+  );
+};
+interface User {
+  name: string;
+  email: string;
+  image: string;
+  phoneNumber: string;
+}
+
+const UserDashboard: React.FunctionComponent = () => {
+  const [users, setUsers] = useState<User[]>();
+
+  const getUsers = async (pageNumber: number): Promise<void> => {
+    try {
+      const response = await fetch("http://localhost:3000/api/admin/users", {
+        method: "GET",
+        query: pageNumber,
+        headers: { "content-type": "application/json" },
+        redirect: "follow",
+      });
+      const data = await response.json();
+      setUsers(data.users);
+    } catch (err) {
+      throw new Error(err.message);
+    }
+  };
+  const pages = 1;
+  useEffect(() => {
+    getUsers(pages);
+  }, [pages]);
+  return (
+    <div>
+      <div className="flex flex-row justify-between text-sm text-center">
+        <p>Name</p>
```

Comment:
these column headers aren't aligned with the rest of the columns on the request table since you also have to account for the accept and decline buttons - make sure to align them with the rest of the ui!